grafanalib/core.py

```diff
@@ -1,5 +1,4 @@
 """Low-level functions for building Grafana dashboards.
-
```

Comment:
Try to minimize irrelevant changes, like removing a blank line.  They make it harder for people to see what really changed.